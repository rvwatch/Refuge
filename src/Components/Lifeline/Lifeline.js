import React from 'react';

export const Lifeline = () => {
  return (
    <section>
      <a href="tel:555-555-5555">555-555-5555</a>
    </section>
  ); 
};
