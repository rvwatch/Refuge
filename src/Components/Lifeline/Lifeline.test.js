import {Lifeline} from './Lifeline';

describe('Lifeline', () => {
  it('should match the snapshot', () => {
    expect(Lifeline).toMatchSnapshot();
  });
});
