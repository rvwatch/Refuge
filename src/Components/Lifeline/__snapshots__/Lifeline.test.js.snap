// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lifeline should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <a
        href="tel:555-555-5555"
>
        555-555-5555
</a>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "555-555-5555",
        "href": "tel:555-555-5555",
      },
      "ref": null,
      "rendered": "555-555-5555",
      "type": "a",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <a
          href="tel:555-555-5555"
>
          555-555-5555
</a>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "555-555-5555",
          "href": "tel:555-555-5555",
        },
        "ref": null,
        "rendered": "555-555-5555",
        "type": "a",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
